guides/source/security.md

```diff
@@ -29,6 +30,147 @@ The threats against web applications include user account hijacking, bypass of a
 
 In order to develop secure web applications you have to keep up to date on all layers and know your enemies. To keep up to date subscribe to security mailing lists, read security blogs, and make updating and security checks a habit (check the [Additional Resources](#additional-resources) chapter). It is done manually because that's how you find the nasty logical security problems.
 
+Authentication
+--------------
+
+Authentication is often one the first features implemented in a web application. It serves as the foundation for securing user data and is part of most modern web applications.
+
+Rails 8 introduces an authentication generator, which provides a solid starting point for securing your application by only allowing access to verified users.
```

Comment:
```suggestion
Starting with version 8.0, Rails now comes with a default authentication generator, which provides a solid starting point for securing your application by only allowing access to verified users.
```

Reply:
Thinking of how this will read in the future when Rails 8 is old news.

Reply:
For most apps, authentication is probably the first thing to implement.
I don't think there will be many who want to add authentication to a Rails 6.1 app.
Anyone reading this to implement authentication probably has a relatively new app started on Rails 8.0.
So maybe we won't need to mention Rails 8 here?